Problem-1_Design_Hashmap.py

```diff
@@ -0,0 +1,78 @@
+# Time Complexity : O(1) - get_ind, put, get and remove functions. O(n) - find_node function where n is the length of the linked list at that

```

Comment:
put, get and remove all make an internal call to find_node, hence the TC should O(n), not O(1).